include/s3select_oper.h

```diff
@@ -820,42 +812,40 @@ class value
     }
     if (l.is_number() && r.is_number())
     {
-      if (l.type != r.type)
-    {
-      //conversion
-
-      if (l.type == value_En_t::DECIMAL)
-      {
-        l.__val.dbl = __op((double)l.__val.num, r.__val.dbl);
-        l.type = value_En_t::FLOAT;
+      if (l.type != r.type) {
```

Comment:
what actually changed from here to the end of the function?
if its only indentation, please revert. this is difficult to review and does not add value here.

Reply:
The problem was that the indentation was wrong (not just a style issue).
Line 824 started a block with ind' of 4 instead of 6, which affected the rest of the function.

Reply:
as mentioned previously i'm using a utility which arrange those issues. 
**astyle --style=stroustrup --style=break --align-pointer=type --align-reference=type --pad-comma --add-braces --indent=spaces=2**

it will be done on dedicated PR, since it create a lot of differences